challenge_3/rust/deg4uss3r/challenge_3/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use std::vec::Vec;
+use std::collections::HashMap;
+
+pub fn majority_finder(input_vec: Vec<u64>) -> u64 {
```

Comment:
This should *definitely* be `Option<u64>`, since (IIRC) the spec says to do nothing if no majority is found.